cdap-api/src/main/java/co/cask/cdap/api/DatasetConfigurer.java

```diff
@@ -32,13 +32,16 @@
    *
    * @param stream {@link Stream}
    */
+  @Deprecated
   void addStream(Stream stream);
```

Comment:
Missing `@deprecated` javadoc for this method

Reply:
Added.